src/tools/linker/docs/design/constant-propagation-and-branch-removal.md

```diff
@@ -12,7 +12,7 @@ Method can return constant value if its code will always return the same value (
     public bool Is32Bit { get => false; }
 ```
 
-On 64bit platforms the property is compiled with constant value, and ILLInk can determine this. It's also possible to use substitutions to overwrite method's return value to a constant via the [substitutions XML file](../data-formats.md#substitution-format). 
+On 64bit platforms the property is compiled with constant value, and ILLInk can determine this. It's also possible to use substitutions to overwrite method's return value to a constant via the [substitutions XML file](../data-formats.md#substitution-format).
```

Comment:
When we consolidated corefx, coreclr, core-setup and mono into dotnet/runtime, we also moved all the docs into a consolidated location: `runtime/docs/`. It might make sense to move these into the consolidated location as well.

Reply:
You marked this comment as resolved but didn't respond. The proposed docs consolidation doesn't need to happen now (or ever, if others disagree) but I would like to know what you think it about my suggestion.

Reply:
In this PR I actually moved the docs, that's why I marked it as resolved. But a couple of people in the team tried to review the PR and it was difficult due to non-functional changes. So we decided to keep things that don't affect build/test for follow-up. Sorry for the confusion, I opened https://github.com/dotnet/runtime/issues/78052 to track moving the docs